components/brave_wallet/browser/cardano/cardano_create_transaction_task.cc

```diff
@@ -152,6 +151,12 @@ void CardanoCreateTransactionTask::RunSolverForTransaction() {
     return;
   }
 
+  if (!CardanoTransactionSerializer().ValidateMinValue(
+          *solved_transaction->TargetOutput(), *latest_epoch_parameters_)) {
+    StopWithError(WalletInternalErrorMessage());
```

Comment:
Can we add a separate user-readable message for this? Since this is not an internal error, by user-input error.